### DISCUSSION TITLE
v. 6.4: Is there a way to access parent loader in child loader

### DISCUSSION THREAD

**Discussion by tkapitein** [remix-run/react-router#9019] [Q&A] [answered]
For my use case I fetch a bunch of settings inside the loader of the Root route. Based on those settings I want to make decisions on what to fetch in child loaders. I have seen the docs about 'useMatches', but as far as I understand, I would not be able to use that inside a loader (it uses DataRouterContext). Is there a way to access that data fetched with the loader on root, inside a child loader?

**Comment by KaRkY** [comment, marked as answer]
No it's not since all loaders are called in parallel.

**Reply by tkapitein** [reply]
Thank you for your reply, I will have to think of a different way to structure this.
